components/Match/Match.jsx

```diff
@@ -44,9 +50,38 @@ class RequestLayer extends React.Component {
   render() {
     return (
       <div>
-        <MatchTable columns={overviewColumns} />
-        <AbilityUpgradesTable columns={abUpgradeColumns} />
-        <BuildingMap match={this.props.match} loading={this.props.loading} />
+        <div>
+          <div>{`Match ${this.props.match.match_id}`}</div>
+          <div>{this.props.match.radiant_win ? 'Radiant Victory' : 'Dire Victory'}</div>
+          <RaisedButton href={`/request#${this.props.match.match_id}`} label={'Parse Replay'} />
+          <RaisedButton href={this.props.match.replay_url} label={'Download Replay'} />
+          <RaisedButton label={'Jist.tv'} />
+          <RaisedButton label={'DotaCoach'} />
+          <table>
+            <tr>
+              <th>Mode</th>
+              <th>Region</th>
+              <th>Duration</th>
+              <th>Ended</th>
+            </tr>
+            <tr>
+              <td>{this.props.match.game_mode}</td>
+              <td>{this.props.match.region}</td>
+              <td>{this.props.match.duration}</td>
+              <td>{this.props.match.start_time + this.props.match.duration}</td>
+            </tr>
+          </table>
+        </div>
+        <Card>
+          <MatchTable columns={overviewColumns} />
+        </Card>
+        <Card>
+          <AbilityUpgradesTable columns={abUpgradeColumns} />
+        </Card>
+        <Card>
+          <BuildingMap match={this.props.match} loading={this.props.loading} />
+        </Card>
+        <MatchTable columns={benchmarksColumns(this.props.match)} />
       </div>
```

Comment:
you might want to consider doing something like the player pages components in the future once the match tabs are increased, then you can put the header in this matches component and similar to the player header.